### ISSUE TITLE
Full-height button in Cover: move to dropdown and make Fullscreen

### ISSUE THREAD

**Issue by jasmussen** [WordPress/gutenberg#27496]
The Cover block has a cool new "full height" button, which sets its height to 100vh in the sidebar controls, and acts as a convenenient toggle:

![cover](https://user-images.githubusercontent.com/1204802/101139132-61516400-3611-11eb-94ca-8d2e1e55559c.gif)

Upon testing this, it seems like you'd nearly always want to combine the 100vh height with full-wide. And in doing so, it could be grouped along with the other alignments/positions. Like so:

<img width="287" alt="Screenshot 2020-12-02 at 10 44 13" src="https://user-images.githubusercontent.com/1204802/101139224-8645d700-3611-11eb-8c19-6cbe5d99d83b.png">

That would also allow us, separately from this effort, to expand the feature to more blocks than just Cover.

**Comment by ntsekouras** [comment]
>And in doing so, it could be grouped along with the other alignments/positions. Like so:

No strong opinion here but I think we shouldn't combine `fullscreen` with alignment options as they are different things, even if fullscreen and wide could be usually combined.

**Comment by retrofox** [comment]
> > And in doing so, it could be grouped along with the other alignments/positions. Like so:
> 
> No strong opinion here but I think we shouldn't combine `fullscreen` with alignment options as they are different things, even if fullscreen and wide could be usually combined.

Yeah... I have a similar feeling but don't have a strong opinion either. In some way, we're going to lose the ability to combine `full height` with all other alignments.

**Comment by jasmussen** [comment]
Excellent question to bring up and work out here in the thread. 

The point of the issue is that you'll almost never want to have _just full-height_ (and if you do, you can in the sidebar with 100vh). You'll almost always want full height and full width. And you can't do 100vw and vh inside post content. So the only solution to getting actual fullscreen, is to combine full-wide with 100vh.

To me, that means the button could sit right below full-wide and toggle both.

**Comment by paaljoachim** [comment]
Let's see here... thinking out loud.
We have a "Change Alignment" drop down.

Align left - pushes to the left.
Align center - pushes to center.
Align right - pushes to the right.
Wide width - creates an extra wide Cover block.
Full width - pushes to the way to the left and right. 

Extending the Full width we have a Full height toggle. As we have the Full width inside the alignment drop down it seems fitting to also have a Full height alignment in the same drop down. One could then do a multi select and select  to have Full width and a Full height. Like so. (I have made some adjustments to the current icons. To signal full width and full height.)

![Cover-block-Full-width-Full-height](https://user-images.githubusercontent.com/5323259/106361311-903c3b80-631d-11eb-8d8e-66187c978a2f.jpg)

Btw
Another alignment I really would like to add is a default width that is by default always selected. So a user can see the default width is selected and then choose something else. https://github.com/WordPress/gutenberg/issues/20941
I do think we discussed this in another issue a while ago. Arguments were brought up why my suggestion was not a great idea. That is thought a sidetrack to the Full-height button issue here.
